event_mass_validation/wizard/confirm_attendees.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="event_registration_massvalidation_view" model="ir.ui.view">
+        <field name="name">event.registration.massvalidation.form</field>
+        <field name="model">event.registration.massvalidation</field>
+        <field name="arch" type="xml">
+            <form string="Confirm Registrations">
+                <p>
+                Are you sure you want to validate all selected registrations?
```

Comment:
I forgot this, please use the text given by Didier on the card : "Are you sure you want to confirm the registration of the selected attendees?"
Also I suppose the module should be translated at least in french because he did add french translations on the card.